packages/evolution-frontend/src/components/admin/validations/InterviewMap.tsx

```
@@ -1,150 +1,504 @@
+                        border: 'none',
+                        cursor: 'pointer',
+                        display: 'flex',
+                        alignItems: 'center',
+                        justifyContent: 'center',
+                        width: '40px',
+                        height: '40px',
+                        fontSize: '30px',
+                        color: '#333'
+                    }}
+                    title={t('FitBounds')}
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)